setup.cfg

```diff
@@ -0,0 +1,33 @@
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = starfish/_version.py
+versionfile_build = starfish/_version.py
+parentdir_prefix = starfish-
+tag_prefix =
+
+[metadata]
+name = starfish
+summary = Pipelines and pipeline components for the analysis of image-based transcriptomics data
+author = Deep Ganguli
```

Comment:
I guess we should fix this at some time to include more peeps. :)

Reply:
Definitely leaving that up to someone else :smile: